### DISCUSSION TITLE
Unable to load cockpit web page at atll.

### DISCUSSION THREAD

**Discussion by ChillarAnand** [cockpit-project/cockpit#20955] [General]
I have done a fresh install on ubuntu machine using the following command.

```
. /etc/os-release
sudo apt install -t ${VERSION_CODENAME}-backports cockpit
```

There are no errors in journalctl.

<img width="1280" alt="Screenshot 2024-08-29 at 19 04 52" src="https://github.com/user-attachments/assets/db4a6cc3-5379-4f0c-b026-86693a9238b4">

<img width="1280" alt="Screenshot 2024-08-29 at 19 04 40" src="https://github.com/user-attachments/assets/d7bf0906-e0fa-47c0-b60f-d76235b70b54">

**Comment by martinpitt** [comment]
This means that the browser is unable to set up a websocket connection. Do you run Cockpit behind  a reverse proxy? Or is that all just one machine and you go to https://localhost:9090? What browser are you using? Do you see anything in the browser console (Ctrl+Shift+J usually) when you try to log in?

**Reply by ChillarAnand** [reply]
I am not using any proxy yet. 

Since it is a remote server, I am using serveo to tunnel into the server. There are multiple errors in the chrome console.

**Reply by ChillarAnand** [reply]
Thanks, @martinpitt for the help. Looks like it was an issue with config.

Once I added `[WebService]` section with new ip/domain, it started working.

It would be great if cockpit can log an error message in such scenarios.

**Reply by martinpitt** [reply]
Ah, then "serveo" kind of was a reverse proxy. It needs to forward websockets, but looks like you figured that out now.
